games/x.yaml

```
@@ -206,6 +206,26 @@
   - 'https://www.techrescue.org/xboing/images/xb-keys.gif'
   - 'https://www.techrescue.org/xboing/images/xb-scores.gif'
 
+- name: Xchomp
+  originals:
+  - Pac-Man
+  type: clone
+  repo: 'https://ftp.funet.fi/pub/Linux/util/X11/games/action/#:~:text=xchomp'
```

```suggestion
  repo: https://github.com/MurphyMc/xchomp
```
slightly updated repo